File: Python/auth/save_cookies.py
import json
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch()
    context = browser.new_context()
    page = context.new_page()
    page.goto("https://example.com")
    context.storage_state(path="cookies.json")
    browser.close()

File: NodeJS/auth/save_cookies.js
const fs = require('fs');
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch();
  const context = await browser.newContext();
  const page = await context.newPage();

  await page.goto('https://example.com');
  const storage = await context.storageState();
  fs.writeFileSync('cookies.json', JSON.stringify(storage, null, 2));

  await browser.close();
})();
